src/templates/homepage.html

```diff
@@ -8,6 +9,29 @@
   <p class="desktop-promo-separator-text">{{ _('More to Explore') }}</p>
 {% endif %}
 
+{% defer (url=anonApi('search')|urlparams(limit=11, doc_type='website'), pluck='objects', id='feed-websites') %}
+  {% if this %}
+    <div class="feed feed-items feed-home c" data-columns>
+    {% if this.length >= 3 %}
+      {{ website_tabs(this|sliceArray(0, 3)) }}
+    {% endif %}
+    {% if this.length >= 6 %}
+      {{ website_carousel(this|sliceArray(3, 12)) }}
+    {% endif %}
+    </div>
+  {% endif %}
+{% placeholder %}
+  <p class="spinner padded alt"></p>
+{% except %}
+  <p class="no-results">
+    {% if error == 400 or error == 404 %}
```

Comment:
Yeah, see this comment: https://github.com/mozilla/fireplace/pull/1419/files#r36241357

I just copy/pasted the regular feed block and changed the endpoint for ease of development. Soon I'll loop around and fix up the feed endpoint to return the websites with it, so it will be part of the regular defer block.